.travis.yml

```diff
@@ -28,7 +29,6 @@ addons:
     - libboost1.55-all-dev
     - libsndfile1-dev
     - libqt4-dev
-    - libgtest-dev
```

Comment:
I don't understand your advice... Commit 7b505bb explains why we cannot use libgtest-dev package to build our unit tests.